physh/ml_physh_map.c

```diff
@@ -7,19 +7,10 @@
 #include <caml/memory.h>
 #include <caml/mlvalues.h>
 #include <caml/fail.h>
+#include <caml/gc_ctrl.h>
```

Comment:
One more conditional is needed as `gc_ctrl.h` is not available in OCaml 4.03.0
Alternatively I could stop supporting this version... It is tempting :P

Reply:
All green now